document_quick_access_folder_auto_classification/security/ir.model.access.csv

```diff
@@ -1,2 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
 access_document_quick_access_missing,access_document_quick_access_missing,model_document_quick_access_missing,group_missing_document,1,1,1,0
+access_document_quick_access_missing_assign,access_document_quick_access_missing_assign,model_document_quick_access_missing_assign,base.group_user,1,0,0,0
```

Comment:
With this permissions, a standard user might not be able to do it. Can you check it please?